core/src/test/java/nl/nn/adapterframework/util/FileUtilsTest.java

```diff
@@ -0,0 +1,283 @@
+package nl.nn.adapterframework.util;
+
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * FileUtils Tester.
+ *
+ * @author <Sina Sen>
+ */
+public class FileUtilsTest {
+
+	@ClassRule
+	public static TemporaryFolder testFolderSource = new TemporaryFolder();
+
+	@ClassRule
+	public static TemporaryFolder testFolderDest = new TemporaryFolder();
+
+	public static String sourceFolderPath;
+
+	public static String destFolderPath;
+
+	private static File f1;
+
+	public String sep = File.separator;
```

Comment:
Since this is a field that doesn't change, I recommend you turning it in to a `public static final String` and with final fields it's a java standard to use all caps as field name, so for example `public static final String FILE_SEPARATOR = File.separator` Not sure if it will be any shorter then `File.separator` which I believe was the idea here but just my 2c.